### ISSUE TITLE
 Easier ai control and upload board carriage as an engi borg

### ISSUE THREAD

**Issue by Yogbot-13** [yogstation13/Yogstation#19671]
## Test Merges: 
Not Supplied / None
## Reproduction:
Description: You can pick up both boards with the engineer borg gripper (Can only attach one image)
## Supplied Image:

![SuppliedImage](https://cdn.discordapp.com/attachments/773277250090434640/1129543574682357790/image.png)
 Submitted by: nickie8995 (842223373789691913)

**Comment by Runian** [comment]
Non-bug/Intentional. Gripper can pick up any board and that means AI Upload boards too.

**Comment by Oblisk234** [comment]
Intentional, per discussion with submitter.
